### PR TITLE
TravelLog endpoints and respective endpoints created

### PR DESCRIPTION
**Pull request by demakk**

### CODE REVIEW

File: Backend/JourneyLog.PL/Controllers/TravelLogPlaceNotePhotoController.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using JourneyLog.BLL.Models.TravelLogPlaceNotePhotos;
+using Microsoft.AspNetCore.Mvc;
+
+namespace JourneyLog.PL.Controllers;
+
+[ApiController]
+[Route("api/travelLog")]
+public class TravelLogPlaceNotePhotoController : ControllerBase
```
Comment (NMishchenko) [inline Backend/JourneyLog.PL/Controllers/TravelLogPlaceNotePhotoController.cs]:
```suggestion
public class NotePhotoController : ControllerBase
```

File: Backend/JourneyLog.PL/Controllers/TravelLogPlaceNoteController.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using JourneyLog.BLL.Models.TravelLogPlaceNote;
+using Microsoft.AspNetCore.Mvc;
+
+namespace JourneyLog.PL.Controllers;
+
+[ApiController]
+[Route("api/travelLog")]
+public class TravelLogPlaceNoteController : ControllerBase
```
Comment (NMishchenko) [inline Backend/JourneyLog.PL/Controllers/TravelLogPlaceNoteController.cs]:
```suggestion
public class NoteController : ControllerBase
```

File: Backend/JourneyLog.BLL/Models/TravelLogPlaceNotePhotos/CreateTravelLogPlaceNotePhotoModel.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿namespace JourneyLog.BLL.Models.TravelLogPlaceNotePhotos;
+
+public class CreateNotePhotoModel
+{
+    public string imageBase64 { get; set; }
```
Comment (NMishchenko) [inline Backend/JourneyLog.BLL/Models/TravelLogPlaceNotePhotos/CreateTravelLogPlaceNotePhotoModel.cs]:
```suggestion
    public string ImageBase64 { get; set; }
```

**Review by NMishchenko — CHANGES_REQUESTED** [review]
Add parameters for path variables

File: Backend/JourneyLog.PL/Controllers/TravelLogPlaceNotePhotoController.cs (hunk 1 above)
Comment (demakk) [reply Backend/JourneyLog.PL/Controllers/TravelLogPlaceNotePhotoController.cs]: Done

File: Backend/JourneyLog.PL/Controllers/TravelLogPlaceNoteController.cs (hunk 2 above)
Comment (demakk) [reply Backend/JourneyLog.PL/Controllers/TravelLogPlaceNoteController.cs]: Done

File: Backend/JourneyLog.BLL/Models/TravelLogPlaceNotePhotos/CreateTravelLogPlaceNotePhotoModel.cs (hunk 3 above)
Comment (demakk) [reply Backend/JourneyLog.BLL/Models/TravelLogPlaceNotePhotos/CreateTravelLogPlaceNotePhotoModel.cs]: Done

**Review by NMishchenko — APPROVED** [review]

**Pull request merged**